### DISCUSSION TITLE
Commercial e-drum modules which support positional sensing

### DISCUSSION THREAD

**Discussion by corrados** [corrados/edrumulus#57] [General]
I just checked at Thomann which is the cheapest e-drum module which supports positional sensing. According to the [this Wiki page](https://en.wikipedia.org/wiki/Roland_V-Drums), only T-27 and TD-50 are supporting positional sensing where the current prices are 1359 € or 2499 € respectively. This is very expensive...
Do you know any other e-drum manufacturer which supports positional sensing? Are the modules available which are cheaper than this?

**Comment by ignotus666** [comment]
The [eDRUMin](https://www.audiofront.net/eDrumIn.php). It's a trigger-MIDI module with no sounds that I _think_ is based on a teensy. The MegaDrum can also do PS but I don't know if it counts as a commercial module (you can DIY it but also purchase complete modules).

**Comment by corrados** [comment]
Shure, eDRUMin does support it. But as I know, it is a "one man show". Aren't there big companies which support this? E.g.:

- Alesis STRIKE PRO SPECIAL EDITION: 
- Yamaha DTX10
- Pearl e/MERGE, Mimic Pro
- 2 Box Drumit Five MKII
- EFNOTE 7X

If you look at the promotional pages of these high end kits, no mentioning of positional sensing at all...

Maybe nobody of the musicians care much about this feature (which I actually cannot understand). Maybe it is no real business case for the companies.

**Comment by ignotus666** [comment]
I think the Mimic Pro recently got an update and now supports PS. My understanding is that the main issue was the patent, held by Roland. But I think that patent has expired now, so others are starting to offer it.

**Comment by corrados** [comment]
That is good news that not only Roland supports this. The bad news is that the Mimic Pro costs 2660 € right now. I am wondering how long it will take until Millenium or Fame will also support this.
